### PR TITLE
Ddpb 1474

### PR DESCRIPTION
**Pull request by elvisciotti**

### CODE REVIEW

File: src/AppBundle/Entity/Report/ReportSubmission.php [hunk 1]
```diff
@@ -113,6 +117,25 @@ public function setArchivedBy($archivedBy)
         return $this;
     }
 
+    /**
+     * @return mixed
```
Comment (slizzio) [inline src/AppBundle/Entity/Report/ReportSubmission.php]: bool?

File: src/AppBundle/Entity/Report/ReportSubmission.php [hunk 2]
```diff
@@ -113,6 +117,25 @@ public function setArchivedBy($archivedBy)
...
+     */
+    public function getDownloadable()
```
Comment (slizzio) [inline src/AppBundle/Entity/Report/ReportSubmission.php]: Maybe call this isDownloadable to be consistent with other boolean properties?

File: src/AppBundle/Entity/Report/ReportSubmission.php [hunk 3]
```diff
@@ -113,6 +117,25 @@ public function setArchivedBy($archivedBy)
...
+    {
+        return $this->downloadable;
+    }
+
+    /**
+     * @param mixed $downloadable
```
Comment (slizzio) [inline src/AppBundle/Entity/Report/ReportSubmission.php]: boolean?

File: src/AppBundle/Entity/Report/Traits/ReportDebtsTrait.php [hunk 4]
```diff
@@ -111,7 +114,9 @@ public function getHasDebts()
     }
 
     /**
-     * @param string $hasDebts
+     * @param $hasDebts bool
```
Comment (slizzio) [inline src/AppBundle/Entity/Report/Traits/ReportDebtsTrait.php]: bool $hasDebts (order)

File: src/AppBundle/Resources/views/Admin/ReportSubmission/index.html.twig [hunk 5]
```diff
@@ -146,7 +146,9 @@
                             </td>
                         {% endif %}
                         <td class="width-tenth change-answer">
-                            <a href="{{ path('admin_documents_download', {'reportSubmissionId': reportSubmission.id}) }}" class="behat-link-download">Download</a><br>
+                            {% if reportSubmission.downloadable %}
```
Comment (slizzio) [inline src/AppBundle/Resources/views/Admin/ReportSubmission/index.html.twig]: isDownloadable?

Comment (elvisciotti) [reply src/AppBundle/Resources/views/Admin/ReportSubmission/index.html.twig]: twig auto-finds the method getDownloadable or isDownloadable

**Pull request merged**